Resources/Prototypes/Entities/Objects/Weapons/Melee/cane.yml

```diff
@@ -0,0 +1,101 @@
+- type: entity
+  parent: BaseItem
+  id: Cane
+  name: cane
+  description: A wooden cane.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/cane.rsi
+    state: cane
+  - type: Item
+    size: Normal
+    sprite: Objects/Weapons/Melee/cane.rsi
+  - type: Appearance
+  - type: MeleeWeapon
+    wideAnimationRotation: 45
+    damage:
+      types:
+        Blunt: 5
+  - type: StaminaDamageOnHit
+    damage: 5
+  - type: Wieldable
+  - type: IncreaseDamageOnWield
+    damage:
+      types:
+        Blunt: 3
+  - type: UseDelay
+    delay: 1
+
+- type: entity
+  name: cane blade
+  parent: BaseItem
+  id: CaneBlade
+  description: A sharp blade with a cane shaped hilt.
+  components:
+  - type: Sharp
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/cane_blade.rsi
+    state: icon
+  - type: MeleeWeapon
+    wideAnimationRotation: 65
+    attackRate: 1.2
+    damage:
+      types:
+        Slash: 12
+    soundHit:
+        path: /Audio/Weapons/bladeslice.ogg
+  - type: Reflect
+    enabled: true
+    reflectProb: .3
+    spread: 90
```

Comment:
Shouldn't be reflecting.

Reply:
shouldn't it be? Its more tc than an e-dagger, and pretty meh even with the reflect.

Reply:
Energy weapons reflecting is intuitive but random bladed weapons doing it is not.

Reply:
Fair